app/front-end/src/features/editor/components/toolbarView/toolbarGroupParams/applyGroupParams.tsx

```diff
@@ -60,115 +57,94 @@ export const ApplyGroupParams: React.FC<ApplyGroupParamsProps> = () => {
   }, [fileTree]);
 
   return (
-    <Box sx={{ display: 'grid', gridTemplateColumns: '50% 50%', p: '1rem' }}>
-      <Box
-        sx={{
-          height: '100%',
-          display: 'grid',
-          flexDirection: 'column',
-          alignItems: 'start',
-          gap: '1rem',
-        }}
-      >
-        <FormControl sx={{ width: '90%' }} size='small'>
-          <GroupParamsInputLabel label={'Apply To'} error={applyError} />
-          <StyledGroupParamsSelect
-            id={'lovd-file-select'}
-            name={'lovd-file-select'}
-            label={'Lovd File'}
-            value={applyToValue}
-            onChange={handleApplyToChange}
-            error={Boolean(applyError)}
-            disabled={blocked}
-            MenuProps={{
-              PaperProps: {
-                sx: {
-                  maxWidth: '15.7rem',
-                  maxHeight: '20rem',
-                  bgcolor: Theme.palette.background.default,
-                },
-              },
-            }}
-          >
-            <StyledGroupParamsListSubheader key={'root'}>
-              <StyledGroupParamsMenuItemTypographyBold>root:</StyledGroupParamsMenuItemTypographyBold>
-            </StyledGroupParamsListSubheader>
-            {fileArray.map((file) => {
-              if (file.type === FileTypes.FOLDER) {
-                return (
-                  <StyledGroupParamsListSubheader key={file.id}>
-                    <StyledGroupParamsMenuItemTypographyBold>{file.id}:</StyledGroupParamsMenuItemTypographyBold>
-                  </StyledGroupParamsListSubheader>
-                );
-              }
-
-              return (
-                <StyledGroupParamsMenuItem key={file.id} value={file.id}>
-                  <StyledGroupParamsMenuItemTypography>{file.label}</StyledGroupParamsMenuItemTypography>
-                </StyledGroupParamsMenuItem>
-              );
-            })}
-          </StyledGroupParamsSelect>
-        </FormControl>
+    <Box sx={{ display: 'flex', flexDirection: 'row', columnGap: '1rem', p: '1rem'}}>
+      <Box sx={{ display: 'flex', flexDirection: 'column', rowGap: '1rem', width: '50%'}}>
+        <Autocomplete
+          size="small"
+          sx={(theme) => ({
```

Comment:
Could have used useTheme() to be consistant with the project, but this works too